### PR TITLE
Fix copyright for unsplash images

### PR DESCRIPTION
**Pull request by ziproot**
**Bugfix:** This PR addresses an issue found in #9355 and #9458.

## Fix Details
Before June 2017, Unsplash was a collection of photos donated to the public domain. On June 2017, Unsplash modified their terms so that all images uploaded after that point were subject to a permissive license, called the Unsplash License. The copyright file did not take this into account. I have updated the copyright file to do so. I have also modified a few other comments to be prefixed with "Comment:" to match the other sections.

UPDATE: I have added a few other changes since I wrote this description:
-I swapped out one of the pictures with one from pixabay because I couldn't find the one on unsplash and this way I know what the license is (the picture on pixabay looks very similar to the one from unsplash and I have made a high DPI pr with this version being a scaled down version of the high DPI one)
-I swapped out two pictures with ones from pixwizard for similar reasons (they are essentially the same image and you would have to know they were changed to see any differences)
-I did reverse image searches of all unsplash images with ambiguous licenses, found their licenses, and recorded them along with archive links to the pictures in the copyright file to verify the license
-The exception is two unsplash images that I think were removed from unsplash and I cannot find on the internet
-I added a clarification that images uploaded after February 2018 have an additional restriction of not being allowed to sell unaltered copies

## Testing Done
I looked at how unsplash changed their terms and conditions, when the images were added to the game, and how other websites, such as Wikipedia, handled this problem.
EDIT: I also looked in game to see how the replaced pixabay image looked.

## Save File
EDIT: I initially didn't have a save file because this only modified the copyright file, but I now have one so you can see the new water4 image, which appears to only be used for the planet "Zenith:"
[Endless-Sky any%~Zenith.txt](https://github.com/endless-sky/endless-sky/files/13110166/Endless-Sky.any.Zenith.txt)
The other images are essentially indistinguishable from the old version and so do not have save files.

### CODE REVIEW

**Comment by ziproot** [comment]
OK, copyright/style checks are failing. Let me see why.

**Comment by ziproot** [comment]
Still broken, and a not helpful error message.

**Comment by ziproot** [comment]
I looked at the document explaining the syntax and I still cannot see where I messed up: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/

**Comment by ziproot** [comment]
There we go.

**Comment by bene-dictator** [comment]
Remember to update the copyright document in the 2x plugin repo.

**Comment by ziproot** [comment]
> Remember to update the copyright document in the 2x plugin repo.

Is that not the high dpi plugin?

**Comment by ziproot** [comment]
I updated badlands 1, badlands 2, and badlands 6 by finding the original images and looking at the upload date. This involved a combination of google reverse image search and searching "site:unsplash.com before:2017-09-15 [keywords related to image]" as those images were added to the game on 9/15/2017. That commit added 26 images, of which I have now done three. It will be harder to do the ones added later, but those are less likely to be public domain.

**Comment by ziproot** [comment]
Converting to draft so I can look through the images in https://github.com/endless-sky/endless-sky/commit/35eb0e760f2763b0ea453c3ce8aa84210fe2a387.

**Comment by ziproot** [comment]
The author of the city11 photo tried to remove it from Unsplash, but it was archived before then. The link to the image on the archive is broken and the published date can't be seen if you just look at the site. You MUST click view source and search for "datePublished" and you will see it was published in August 2017. You will also see in the source that the image is "https://web.archive.org/web/20221105085409im_/https://images.unsplash.com/photo-1501607087079-27057d2fdc93" which matches the image. The archive has a bunch of extra tags that break the image. Additionally, as of 10/20/2023, the image file itself is still on Unsplash, even if you can't find anything else. The internet never forgets.

**Comment by ziproot** [comment]
I'm going to swap out an image with a duplicate on pixabay because I can't find the unsplash one.

**Comment by ziproot** [comment]
OK, I have sorted all of the images from unsplash in commit https://github.com/endless-sky/endless-sky/commit/35eb0e760f2763b0ea453c3ce8aa84210fe2a387: 
[advanced copyright.txt](https://github.com/endless-sky/endless-sky/files/13058810/advanced.copyright.txt)

**Comment by ziproot** [comment]
There are still plenty of images that have not been verified, but those were added well after the license change and I'm assuming most, if not all, of them are unsplash license.

**Comment by ziproot** [comment]
Updated description and added a save file to see Zenith.

**Comment by Saugia** [comment]
If we're going to be doing this with other similar cases, we should probably have links to those images as well, including additions in this PR that don't have links to the images. Are those just oversights, out of scope, or hard to source? Either way we probably will end up wanting to find the links anyway, just curious.

**Comment by ziproot** [comment]
> If we're going to be doing this with other similar cases, we should probably have links to those images as well, including additions in this PR that don't have links to the images. Are those just oversights, out of scope, or hard to source? Either way we probably will end up wanting to find the links anyway, just curious.

Are you talking about the other unsplash images? My main motivation for this PR was to see what images would actually need to be replaced if we decided we didn't like the unsplash license. As there were several images that were added just after the license change, some of them were likely to be public domain, so I went through those individually to see which ones could be saved in the event that we were to remove all images with the unsplash license. The other images I didn't look at were added several years after the license change and are unlikely to be public domain, so I didn't bother to try to reverse image search hundreds of images to find three of them that were public domain and wouldn't have to be replaced.

The only "addition" in the form of an image that wasn't already in-game is the new water4 image, and I provided a link to the image there. If by "other similar cases" you mean the ones from morguefile.com, those would be oversights as I just now noticed that they could be a problem, but that's outside the scope of this PR as those aren't unsplash images. (Also, the morgue file license actually seems fine as it allows you to sell, distribute, etc. so long as you provide credit).

**Comment by Saugia** [comment]
I'm referring to images that were touched up in this PR that don't have links to them. If we're looking to eventually find links to all images necessary, it would also include those unsplash ones that don't currently have links, so we figure that it can also be done in this PR if we're going to do so anyway.

**Comment by ziproot** [comment]
> I'm referring to (unsplash) images that were touched up in this PR that don't have links to them. If we're looking to eventually find links to all images necessary, it would also include those unsplash ones that don't currently have links, so we figure that it can also be done in this PR if we're going to do so anyway.

OK. I will mark this as a draft. It may take a while, but I will try to find links for every unsplash image we use.

**Comment by ziproot** [comment]
I just found sources for 11 images. Four of them are public domain. I severely underestimated the number of unsplash images added to the game well after the change to unsplash license that were still public domain. That is my fault. I am going to try to find another 10 or so each day. At that pace I should be done before next release.

**Comment by bene-dictator** [comment]
I added some of those, if you need some help I can get help the images links for you. 

FYI:
- https://github.com/endless-sky/endless-sky/pull/7764
- https://github.com/endless-sky/endless-sky/pull/7582

**Comment by ziproot** [comment]
Thank you, that would be very helpful.

**Comment by bene-dictator** [comment]
badlands13: https://unsplash.com/photos/tRfYcGc6sFk
beach15: https://unsplash.com/photos/DqpKPaKNxjw
city19: https://unsplash.com/photos/WNVGBGqVQeY
city20: https://unsplash.com/photos/pxrn4Z671BI
city21: https://unsplash.com/photos/oxhHjLKhj-4
city22: https://unsplash.com/photos/KKk1f8hMFkE
city23: https://unsplash.com/photos/Y9Wp5keJsn8
city24: https://unsplash.com/photos/yqWOgqmwP0M
city25: https://unsplash.com/photos/aM0Y1dQtMg0
desert14: https://unsplash.com/photos/KW4SGE0Irus
desert15: https://unsplash.com/photos/pZ-XFIrJMtE
forest10: https://unsplash.com/photos/9pF2dPx5H4A
mountain27: https://unsplash.com/photos/y-VhDt6KsKg
mountain28: https://unsplash.com/photos/8Nql6oVT_1A
mountain29: https://unsplash.com/photos/PdWe5FyhpRI
mountain30: https://unsplash.com/photos/lSh6vd87jTs
sea22: https://unsplash.com/photos/r3peJAPApQw
snow22: https://unsplash.com/photos/P81LbTij5Ro

**Comment by ziproot** [comment]
Wayback machine has rate limited me, so it may take some time to get all the images archived. Those links as they stand aren't stable, and stable links won't work if the image gets deleted, so I put it to the wayback machine, then shorten it with v.gd so people can see where it's going.

**Comment by bene-dictator** [comment]
Do the long links help?

**Comment by ziproot** [comment]
Not really, as they aren't archived. The links were helpful, though.

**Comment by ziproot** [comment]
OK, I have updated the copyrights for all of the unsplash images bene-dictator linked to. Thanks again for your help.

**Comment by ziproot** [comment]
station43 was deleted from unsplash six months ago. By deleted I mean the information about who made the image was deleted by the uploader six months ago. The image itself is [still on unsplash's servers](https://images.unsplash.com/photo-1544828184-2526634b2fa1?crop=entropy&cs=tinysrgb&fit=max&fm=jpg&ixid=MnwxMjA3fDB8MXxhbGx8fHx8fHx8fHwxNjI3MjU2NDAw&ixlib=rb-1.2.1&q=80&w=1080). There is an archived version of the same image on Pexels, which is subject to the Pexels license. Should I switch to that?

**Comment by ziproot** [comment]
I have sourced everything that had ambiguous copyright, other than the unused portraits. I am splitting this into several pull request because I think this is the most important. Once this gets merged I will work on the next pull request, which will be the portrait unsplash images and the unsplash images added to the game before the license change, and then there will be a third pull request for the other non-unsplash images that were taken from the internet.

**Comment by ziproot** [comment]
@Saugia how does this look?

File: copyright [hunk 1]
```diff
@@ -396,9 +413,13 @@ Comment: Derived from works by Iaz Poolar (under the same license).
...
-License: public-domain
- Taken from unsplash.com, a collection of photographs that have been donated and
- placed in the public domain.
+License: Depends
+Comment:
+ Taken from unsplash.com and added to the game after June 2017. If the pictures were
+ uploaded to unsplash.com before June 2017, they were donated and placed in the public
+ domain. If the pictures were uploaded to unsplash.com on or after June 2017, they are
+ subject to the Unsplash License. If they were also uploaded after February 2018, they
+ are subject to additional restrictions.
```
[the remark this answers is not in the document]
Comment (ziproot) [reply copyright:422]: This is to say I haven't reviewed the portraits because they aren't used in game and I'm planning to do that on a later PR.

**Comment by ziproot** [comment]
> Is v.gd a link shortener? I'd rather not have those in here.

All right, I'll change it to archive.is.

**Review by warp-core — APPROVED** [review]

**Pull request merged**